source/api_documentation/network_integration/advertiser_campaigns/index.rst

```diff
@@ -386,6 +388,10 @@ Parameter Details
     - Type
     - Value
 
+  * - asr_phrases
+    - Array of strings
+    - A list of phrases, where each phrase correlates to a keypress in the Menu node. Can only be used when speech recognition is enabled. For example, a value of ["sales", "support"] would designate phrase "sales" for keypresses 1 and phrase "support" for keypress 2. Using asr_phrases allows the caller to respond verbally with one of the configured phrases instead of making a keypress.
```

Comment:
This doesn't seem to match the demo screenshot because this says to submit `["sales", "support"]`, but the screenshot says to submit `[{ "phrase": "sales" }, { "phrase": "support" }]`.

Reply:
Again, totally forgot to push my changes up 🤦‍♀️ they're here now!